pyston/converter/__init__.py

```diff
@@ -211,32 +238,23 @@ def render_value(self, value, first=True):
     def _get_value_from_row(self, data, field):
         return self.render_value(self._get_recursive_value_from_row(data, field.key_path) or '')
 
-    def _render_content(self, field_name_list, converted_data):
-        result = []
+    def _render_row(self, row, field_name_list):
+        return (self._get_value_from_row(row, field) for field in field_name_list)
 
+    def _render_content(self, field_name_list, converted_data):
         constructed_data = converted_data
-        if not isinstance(constructed_data, (list, tuple, set)):
+        if not isinstance(constructed_data, (list, tuple, set, types.GeneratorType)):
             constructed_data = [constructed_data]
 
-        for row in constructed_data:
-            out_row = []
-            for field in field_name_list:
-                out_row.append(self._get_value_from_row(row, field))
-            result.append(out_row)
-        return result
-
-    def _get_output(self):
-        return BytesIO()
+        return (self._render_row(row, field_name_list) for row in constructed_data)
```

Comment:
tady musím vracet opět generátor jinak by to celé nemělo smysl :) Navíc to i vypadá lépe ne? row musím renderovat přes funkci jelikož je to opět generátor